modules/zero-trust-manager-spire-agent-config.adoc

```diff
@@ -24,25 +25,47 @@ You can configure the `SpireAgent` custom resource (CR) to deploy and configure
 +
 [source,yaml]
 ----
-apiVersion: operator.openshift.io/v1alpha1
+apapiVersion: operator.openshift.io/v1alpha1
 kind: SpireAgent
 metadata:
   name: cluster
 spec:
-  trustDomain: <trust_domain> #<1>
-  clusterName: <cluster_name> #<2>
+  socketPath: "/run/spire/agent-sockets"
+  logLevel: "info"
+  logFormat: "text"
   nodeAttestor:
-    k8sPSATEnabled: "true" #<3>
+    k8sPSATEnabled: "true"
   workloadAttestors:
-    k8sEnabled: "true" #<4>
+    k8sEnabled: "true"
     workloadAttestorsVerification:
-      type: "auto" #<5>
+      type: "auto"
+      hostCertBasePath: "/etc/kubernetes"
+      hostCertFileName: "kubelet-ca.crt"
+    disableContainerSelectors: "false"
+    useNewContainerLocator: "true"
 ----
-<1> The trust domain to be used for the SPIFFE identifiers.
-<2> The name of your cluster.
-<3> Enable or disable the projected service account token (PSAT) Kubernetes node attestor. The valid options are `true` and `false`.
-<4> Enable or disable the Kubernetes workload attestor. The valid options are `true` and `false`.
-<5> The type of verification to be done against the kubelet. Valid options are `auto`, `hostCert`, `apiServerCA`, `skip`. The `auto` option initially attempts to use `hostCert`, and then falls back to `apiServerCA`.
+
+name:: Must be named 'cluster'.
+
+socketPath:: The directory on the host where the SPIRE agent socket is created. This directory is shared with the SPIFFE CSI driver via the `hostPath` volume. Must match the `SpiffeCSIDriver.spec.agentSocketPath` for workloads to access the socket. Must be an absolute path with a maximum length of 256 characters.
+
+logLevel:: The logging level for the SPIRE Server. The valid options are `debug`, `info`, `warn`, and `error`.
+
+logFormat:: The logging format for the SPIRE Server. The valid options are `text` and `json`.
+
+k8sPSATEnabled:: Specifies whether Kubernetes Projected Service Account Token (PSAT) node attestation is enabled. When enabled, the SPIRE agent uses K8s PSATs to prove its identity to the SPIRE server during node attestation. The valid options are `true` and `false`.
+
+k8sEnabled:: Specifies whether the Kubernetes workload attestor is enabled. When enabled, the SPIRE agent can verify workload identities using Kubernetes pod information and service account tokens. The valid options are `true` and `false`.
+
+type:: The kubelet certificate verification mode. The valid options are `auto` (verify using OpenShift defaults), `hostCert` (use a custom CA certificate), and `skip` (skip TLS verification entirely).
+
+hostCertBasePath:: The directory containing the kubelet CA certificate. Required when type is `hostCert`. Optional when type is `auto` (defaults to /etc/kubernetes if not specified).
```

Comment:
Out of scope for merge review, but be careful about parentheses. It goes against style guidelines to use them in most cases, esp. like this. Also take care to try to mark up in backticks things like file names/locations/values. I see them hit or miss throughout the PR.